src/terminal/adapter/adaptDispatchGraphics.cpp

```diff
@@ -261,6 +279,27 @@ void AdaptDispatch::_SetGraphicsOptionHelper(const DispatchTypes::GraphicsOption
     }
 }
 
+// Routine Description:
+// Returns true if the GraphicsOption represents an extended text attribute.
+//   These include things such as Underlined, Italics, Blinking, etc.
+// Return Value:
+// - true if the opt is the indicator for an extended text attribute, false otherwise.
+bool AdaptDispatch::s_IsExtendedTextAttribute(const DispatchTypes::GraphicsOptions opt) noexcept
+{
+    // TODO:GH#2916 add support for DoublyUnderlined, Faint(RGBColorOrFaint).
+    // These two are currently partially implemented as other things:
+    // * Faint is kinda like the opposite of what bold does

```

Comment:
```suggestion
    // * Faint is approximately what the opposite of what bold does
```

"kinda like"... ehhhhhhhhh